dask/array/tests/test_fft.py

```diff
@@ -108,3 +113,41 @@ def test_fft_consistent_names():
     assert same_keys(fft(darr, 5), fft(darr, 5))
     assert same_keys(fft(darr2, 5, axis=0), fft(darr2, 5, axis=0))
     assert not same_keys(fft(darr, 5), fft(darr, 13))
+
+
+def test_wrap_numpy():
```

Comment:
Also I have a slight preference to use names like `x` and `dx` rather than `nparr`.  These names were probably from before, but this refactor might be a nice chance to rename them.